deploy/operator/00_scylladbmonitoring_prometheus_clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scylladb:monitoring:prometheus
```

Comment:
i don't see it being used anywhere, what's the purpose of it?

Reply:
it's referenced in the prometheus rolebinding
https://github.com/prometheus-operator/prometheus-operator/blob/8cebe1b49dda14b26dd42863d4d5842751844029/Documentation/rbac.md#prometheus-rbac